apps/src/appMain.js

```diff
@@ -92,7 +92,7 @@ export default function (app, levels, options) {
           blockInstallOptions,
           customBlocksConfig,
           options.level,
-          level.hideCustomBlocks,
+          level.hideCustomBlocks && !options.level.editBlocks,
```

Comment:
isn't it `edit_blocks`?

Reply:
...it would appear that both of them are set to the same thing. `edit_blocks` is used much more often though.

Reply:
Gross. https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/StudioApp.js#L2422